mealie/routes/households/controller_invitations.py

```diff
@@ -21,18 +21,31 @@
 class GroupInvitationsController(BaseUserController):
     @router.get("", response_model=list[ReadInviteToken])
     def get_invite_tokens(self):
-        return self.repos.group_invite_tokens.page_all(PaginationQuery(page=1, per_page=-1)).items
+        if self.user.admin:
+            return self.repos.all_groups_invite_tokens.page_all(PaginationQuery(page=1, per_page=-1)).items
+        else:
+            return self.repos.group_invite_tokens.page_all(PaginationQuery(page=1, per_page=-1)).items
 
     @router.post("", response_model=ReadInviteToken, status_code=status.HTTP_201_CREATED)
-    def create_invite_token(self, uses: CreateInviteToken):
+    def create_invite_token(self, body: CreateInviteToken):
         if not self.user.can_invite:
             raise HTTPException(
                 status.HTTP_403_FORBIDDEN,
                 detail="User is not allowed to create invite tokens",
             )
 
+        if body.group_id and body.household_id and not self.user.admin:
```

Comment:
Can we modify this to compare against `self.group_id` and `self.household_id`, rather than just checking if they're provided? That way if a user provides their own group/household it doesn't error out. Something like:

```python
body.group_id = body.group_id or self.group_id
body.household_id = body.household_id or self.household_id

if not self.user.admin and (body.group_id != self.group_id or body.household_id != self.household_id):
    ...
```